### PR TITLE
🐞 fix(chips): Activities pills text being cut

### PR DESCRIPTION
**Pull request by CaioAugustoR**
[PCODE-461](https://gympass.atlassian.net/browse/PCODE-461)

## Description 📄

In current behavior on activity sheets is being cut. in the correction, the text was centered so as not to cut.

## Platforms 📲

- [ ] Web
- [x] Mobile

## Type of change 🔍

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? 🧪

[Enter the tips to test this PR]

- [ ] Unit Test
- [x] Snapshot Test

## Checklist: 🔍

- [ ] My code follows the contribution guide of this project [Contributing Guide](https://github.com/Gympass/yoga/blob/master/CONTRIBUTING.md)
- [ ] Layout matches design prototype: [FIGMA](https://figma.com/file/YOUR_LINK)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Screenshots 📸

[Upload your screenshots here]

| Before | After |
| ------ | ----- |
|        |       |

[PCODE-461]: https://gympass.atlassian.net/browse/PCODE-461?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ericcleao — APPROVED** [review]

**Comment by ericcleao** [comment]
I notice that the wrapper has a fixed height instead of a padding over the text content, what do you thing about change this approach?

**Review by leonardokl — APPROVED** [review]

**Review by eduardobreno — APPROVED** [review]

**Comment by CaioAugustoR** [comment]
> I notice that the wrapper has a fixed height instead of a padding over the text content, what do you thing about change this approach?

So I thought about that too. in putting a min height and padding but in our design system this component should only be 32 pixels high

File: README.md [hunk 1]
```diff
@@ -107,6 +109,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
       <td align="center"><a href="http://guilhermecardoso.dev.br"><img src="https://avatars.githubusercontent.com/u/15979107?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Luis Guilherme Cardoso Rosa</b></sub></a><br /><a href="https://github.com/gympass/yoga/commits?author=lguilhermecardoso" title="Code">💻</a></td>
       <td align="center"><a href="https://github.com/matheushdsbr"><img src="https://avatars.githubusercontent.com/u/32910717?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Matheus Henrique</b></sub></a><br /><a href="https://github.com/gympass/yoga/commits?author=matheushdsbr" title="Code">💻</a></td>
+      <td align="center"><a href="https://github.com/CaioAugustoV"><img src="https://avatars.githubusercontent.com/u/31218003?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Caio Augusto</b></sub></a><br /><a href="https://github.com/gympass/yoga/commits?author=CaioAugustoR" title="Code">💻</a></td>
```
Comment (matheushdsbr) [inline README.md:112]: Yo @CaioAugustoR, thanks for your contribuition! 

I don't know if what you did here will work well, could you remove this change from the readme?

after the merge I can update the contributors

**Review by matheushdsbr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CaioAugustoR) [reply README.md:112]: Ok!

File: README.md [hunk 2]
```diff
@@ -3,7 +3,9 @@
 </p>
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-40-orange.svg?style=flat-square)](#contributors)
+
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
```
Comment (caioalexandrebr) [inline README.md:9]:
```suggestion
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-40-orange.svg?style=flat-square)](#contributors)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

**Review by caioalexandrebr — CHANGES_REQUESTED** [review]

**Review by caioalexandrebr — APPROVED** [review]
LGTM 🚀

**Review by matheushdsbr — APPROVED** [review]
🚀 🚀

**Review by guilhermecardoso-gympass — APPROVED** [review]

**Comment by matheushdsbr** [comment]
@all-contributors please add @CaioAugustoR for bug

**Pull request merged**